packages/pyodide-kernel-extension/package.json

```diff
@@ -47,14 +47,14 @@
     "watch:labextension": "jupyter labextension watch ."
   },
   "dependencies": {
-    "@jupyterlab/coreutils": "^6.1.1",
-    "@jupyterlite/contents": "^0.5.0",
-    "@jupyterlite/kernel": "^0.5.0",
+    "@jupyterlab/coreutils": "^6.4.0-alpha.2",
```

Comment:
Also bumping these `@jupyterlab` packages, since `jupyterlite-pyodide-kernel` already closely follows `jupyterlite-core` releases, and `jupyterlite-core` follows minor JupyterLab releases.

Reply:
Sounds good, but same old question: how can we make this more predictable, and based on the truth from upstream's `yarn.lock` and `package.json`,  and then doubly upstream's `staging`?

Reply:
What kind of predictability are we expecting?

In this case these `@jupyterlab` packages are used more like in regular extensions, and it's mostly minimal use of `@jupyterlab/coreutils` and `@jupyterlab/services`.

Reply:
If we're sticking with `jlpm` as the main task entry point:

```bash
jlpm
jlpm deduplicate
jlpm lint:upstreams
```

... and it checks vs a single source of truth that the not-going-to-space-if-wrong versions from the currently-installed python and js upstreams. It doesn't even have to _fix_ them, just tell us what is wrong.

Reply:
Or, perhaps it would be more appropriate as a pytest which only runs if it's in a git checkout.